test/util/nightwatch-commands/fillMonthYear.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
I made this and then realized I didn't need it right now, but we will later.

Reply:
We could just be able to adapt `.fillDate()` to take a partial, too.